Woodall_Robert/Assignments/multiply.py

```diff
@@ -0,0 +1,18 @@
+'''
+
+Create a function called 'multiply' that reads each value in a list 
+and returns a list where each value has been multiplied by 5.
+
+'''
+
+def multiply(input, multiplier):
+	newList = []
+
+	for index in range(len(input)):
+		newList.append(input[index] * multiplier)
```

Comment:
since you do not care about `index`, it's simpler to iterate directly over the elements:

`for ele in input: newList.append(ele*multiplier)`